### PR TITLE
Sobiech`s branch

### PR DESCRIPTION
**Pull request by SobiechQ**
Classes used for .torrent file reading.

### CODE REVIEW

File: src/main/java/org/SittX/MyBEncoding/MyMain.java
Comment (SittX) [inline src/main/java/org/SittX/MyBEncoding/MyMain.java:1]: It would be better if we have only one main file.

**Review by SittX — APPROVED** [review]
IMO it would be more descriptive to change the class name to a more specific one instead of "My...". Also, I'll clean up my code and integrate your classes into the Main function.

File: src/main/java/org/SittX/MyBEncoding/MyMain.java
Comment (SobiechQ) [reply src/main/java/org/SittX/MyBEncoding/MyMain.java:1]: Agreed. When I'm done with tests I'll just remove it completely. I didn't want to modify your Main but I wanted to run classes to test them

Comment (SobiechQ) [reply src/main/java/org/SittX/MyBEncoding/MyMain.java:1]: Lists are still WIP. After I add them and test them I'll delete the testing main

**Pull request merged**